frontend/public/vendor.scss

```diff
@@ -5,6 +5,7 @@
 @import "~openshift-logos-icon/style";
 @import "~@fortawesome/fontawesome-free/scss/fontawesome";
 @import "~@fortawesome/fontawesome-free/scss/solid";
+@import "~@patternfly/patternfly/patternfly-theme-dark";
```

Comment:
You'll need to import this after all other `@patternfly/patternfly/` import rules, so after  `@import "~@patternfly/patternfly/patternfly-charts";`